src/pages/developing/dev-resources/index.mdx

```diff
@@ -132,6 +132,28 @@ your framework of choice.
   </ResourceCard>
 </Column>
 
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Carbon Storybook Theme Switcher"
+    href="https://www.npmjs.com/package/@carbon/storybook-addon-theme"
+    >
+
+![](/images/storybook-addon-carbon-themes.png)
+
+  </ResourceCard>
+</Column>
+
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Stylelint plugin Carbon tokens"
+    href="https://www.npmjs.com/package/stylelint-plugin-carbon-tokens"
```

Comment:
```suggestion
    href="https://github.com/carbon-design-system/stylelint-plugin-carbon-tokens"
```

would you prefer linking to the repo instead so you can reuse the GitHub icon on this card?